### DISCUSSION TITLE
Progress Report 2 – 6 May

### DISCUSSION THREAD

**Discussion by esheehan-gsl** [NOAA-GSL/unified-graphics#17] [Progress Report]
🚴🏻 **Progress**
- Obtained sample NetCDF diag files, which should be easier to work with than the binary GSI files
- Began setting up tooling for the frontend (#16)

😖 **Regressions**
- None

💬 **To Discuss**
- Configuring a dev environment with Kubernetes

📆 **Next Week**
- Complete set up of frontend tools
- Explore NetCDF diag files

**Comment by ian-noaa** [comment]
🚴🏻 **Progress**
- Submitted infrastructure request to VLab

😖 **Regressions**
- None

💬 **To Discuss**
- Configuring a dev environment with Kubernetes

📆 **Next Week**
- Explore ways (Argo CD, ECR triggers) for the Kubernetes cluster to detect and pull new images created in GitHub Actions
